core/src/main/java/org/bitcoinj/crypto/BLSKeyExchangeCrypter.java

```diff
@@ -103,85 +76,8 @@ public KeyParameter deriveKey(BLSSecretKey secretKey, BLSPublicKey blsPeerPublic
             log.info("Deriving key took {} for a BLS Key Exchange", watch);
             return new KeyParameter(key);
         } catch (Exception e) {
-            throw new KeyCrypterException("Could not generate key from password and salt.", e);
-        }
-    }
-    /**
-     * Password based encryption using AES - CBC 256 bits.
-     */
-    @Override
-    public EncryptedData encrypt(byte[] plainBytes, KeyParameter aesKey) throws KeyCrypterException {
-        checkNotNull(plainBytes);
-        checkNotNull(aesKey);
-
-        try {
-            // Generate iv - each encryption call has a different iv.
-            byte[] iv = new byte[BLOCK_LENGTH];
-            secureRandom.nextBytes(iv);
-
-            ParametersWithIV keyWithIv = new ParametersWithIV(aesKey, iv);
-
-            // Encrypt using AES.
-            BufferedBlockCipher cipher = new PaddedBufferedBlockCipher(new CBCBlockCipher(new AESEngine()));
-            cipher.init(true, keyWithIv);
-            byte[] encryptedBytes = new byte[cipher.getOutputSize(plainBytes.length)];
-            final int length1 = cipher.processBytes(plainBytes, 0, plainBytes.length, encryptedBytes, 0);
-            final int length2 = cipher.doFinal(encryptedBytes, length1);
-
-            return new EncryptedData(iv, Arrays.copyOf(encryptedBytes, length1 + length2));
-        } catch (Exception e) {
```

Comment:
this method (decrypt) and the following encrypt method were refactored into a new class called KeyCrypterAESCBC